### ISSUE TITLE
Do you plan to integrate other product groups?

### ISSUE THREAD

**Issue by whoknows86** [mback2k/pywmspro#1]
such as roller shutters and blinds and the weather data?

**Comment by mback2k** [comment]
Yes, for that I would need the integrations diagnostic output via Home Assistant.

Also see: https://community.home-assistant.io/t/warema-wms-webcontrol-pro-integration/548835/20?u=mback2k

**Comment by whoknows86** [comment]
Du hast ja schon einige logs bekommen, Woche Gerätetypen fehlen dir den noch?
Würde mein webcontrol entsprechend anpassen, damit die logs vll einfacher sind

**Comment by mback2k** [comment]
Ich hatte noch keine Zeit mich mit den eingereichten Logs zu beschäftigen, da ich erst noch ein paar Fehler ausmerzen muss. Kommt aber 👍

**Comment by whoknows86** [comment]
Sorry für die vielleicht doofe Frage, aber in welchen Ordner muss ich dir Diagnostic reinladen, finde keinen wmspro... Suche aber bestimmt an der falschen Stelle😅

**Comment by mback2k** [comment]
Du musst du die Diagnostics nirgendwo reinladen, sondern am besten hier anhängen (per drag & drop). 😉

**Comment by chris9o** [comment]
[config_entry-wmspro-01J9YA5BNVXSZMQ34SG8RZV93V-1.json.txt](https://github.com/user-attachments/files/18269934/config_entry-wmspro-01J9YA5BNVXSZMQ34SG8RZV93V-1.json.txt)

The elements "DACH" (roof top shutter) and both elements "HEIZUNG" (two ir heater) are curently not supported. Would bei glad If you may add them to the awesome integration of WMS into HAS.

Thanks for your great work !

**Comment by Kurdie2023** [comment]
Hi @mback2k,

would it be possible to add support for an awning with volant? That would be really great!

[config_entry-wmspro-01K2PP0BGM5PRQXG1VJ8Z6KX6K.json](https://github.com/user-attachments/files/21795181/config_entry-wmspro-01K2PP0BGM5PRQXG1VJ8Z6KX6K.json)

Thank you very much for your work!

**Comment by mback2k** [comment]
Heaters, Slat-based roofs and covers as well as volants should be supported now or with the upcoming Home Assistant release.
